packages/nimiq-vitepress-theme/src/assets/github-callouts.css

```diff
@@ -1,9 +1,10 @@
 .custom-block.github-alert {
   background: var(--bg-color);
-  --nq-m-min: 14;
-  --nq-m-max: 16;
+  --margin-min-em: 1.5;
+  --margin-max-em: 2;
   padding: var(--nq-m-size);
-  margin: calc(1.5 * var(--nq-m-size)) 0;
+  margin-top: var(--nq-m-size);
+  margin-bottom: var(--nq-m-size);
```

Comment:
In this changed variables from `--nq-m-min` to `--margin-min-em` but the `margin-top` still uses the "old" one?